e2e/wdio/platform/pages/thumbnail.po.ts

```diff
@@ -0,0 +1,19 @@
+import { BaseComponentPo } from './base-component.po';
+import { waitElementToBePresentInDOM, waitForElDisplayed } from '../../driver/wdio';
+
+export class ThumbnailPo extends BaseComponentPo {
+
+    url = '/thumbnail';
+    root = '#page-content';
+
+    mainImage = '.fdp-media-container img';
+    verticalGalleryImages = 'fdp-thumbnail-basic-example fd-avatar';
+    horizontalGalleryImages = 'fdp-thumbnail-horizontal-example fd-avatar';
+    verticalGalleryVideo = 'fdp-thumbnail-video-media-example fd-avatar';
+
+    open(): void {
+        super.open(this.url);
+        waitElementToBePresentInDOM(this.root, 30000);
+        waitForElDisplayed(this.root, 30000);
```

Comment:
let's use Nikita's solution by adding waitForPresent(this.mainImg); to the open() method here and after the refresh in the afterEach hook in the spec

Reply:
renamed waitElementToBePresentInDOM to waitForPresent

Reply:
open(): void {
        super.open(this.url);
        waitForPresent(this.mainImage, 30000);
        waitForElDisplayed(this.mainImage, 30000);